controllers/publishController.js

```diff
@@ -88,36 +88,43 @@ module.exports = {
   },
   claimNameIsAvailable (name) {
     // find any records where the name is used
-    return db.File.findAll({ where: { name } })
```

Comment:
I'm confused by this function, though I would have been equally confused before this changeset. If we're querying our own DB, why do we have to consider just the ones with a certain address? Is that because the DB contains all claims?

Also, if the addresses are in the DB, can't this filtering be done in SQL rather than JS?

Reply:
Yes, the table that is being queried contains all claims, so we have to filter out claims belonging to this address.  I wonder if there would be reason to create a separate table for claims the site owns?  We avoided doing that previously because it seemed to be unnecessarily duplicative.